go.mod

```diff
@@ -34,11 +28,33 @@ require (
 	golang.org/x/crypto v0.0.0-20200302210943-78000ba7a073
 	golang.org/x/oauth2 v0.0.0-20181003184128-c57b0facaced
 	golang.org/x/sys v0.0.0-20211216021012-1d35b9e2eb4e
+	google.golang.org/api v0.0.0-20181016191922-cc9bd73d51b4
+	gopkg.in/DataDog/dd-trace-go.v1 v1.28.0
+)
+
+require (
+	github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d // indirect
```

Comment:
This split happened as a result of running `go mod tidy` using go 1.18 - it splits the direct dependencies up from the indirect ones